doc/general.html

```diff
@@ -1,7 +1,6 @@
 ---
 layout: default
 title: Documentation
-canonical_url: /doc/
```

Comment:
This may need some changes to [CI](https://github.com/neovim/doc/blob/main/ci/doc-index.sh).

Reply:
update DOC_INDEX_PAGE_URL you mean ? do you know why it was set != doc/index.html ?

Reply:
No clue.  Just ran into it while going through the CI scripts.

Reply:
ok I think I will merge this MR soon. I will be on the lookout for failures and update the CI repo accordingly (maybe add a README as well).